contracts/proxy/Clones.sol

```diff
@@ -13,6 +13,11 @@ pragma solidity >=0.6.0 <0.8.0;
  * proposed bytecode. This is possible using both create and create2.
  */
 library Clones {
+    /**
+     * @dev Deploys and return the address of a clone that mimics the behaviour of `master`.
+     *
+     * This function uses the create opcode, which should never revert.
```

Comment:
I just realized that this could potentially revert in an out of gas situation. I guess this means we should revert if `instance` is 0.

Reply:
In out of gas, would the instance be set to 0 (which we should check) ? I was under the impression that the call would just revert on its own